content/events/2020-montreal/contact.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Team / Contact
```

Comment:
Because of the bilingual titles the top bar cuts off (at least for me)
<img width="1280" alt="Screenshot 2020-03-04 at 06 17 17" src="https://user-images.githubusercontent.com/1084449/75874529-e3d99880-5ddf-11ea-8f4c-e9bb2609cfb4.png">

Reply:
Seems to be a general problem with the site design, if I resize the browser the whole bar disappears :( but I'm no front end dev so 🤷‍♂

Reply:
yeah; that does not look great; I'll probably revert back to single language for titles

Reply:
unless I feel creative with css...

Reply:
FYI I'm currently working on a fix for this